sunkit_image/radial.py

```diff
@@ -634,13 +630,14 @@ def _percentile_ranks_numpy_inplace(arr):
 @u.quantity_input(application_radius=u.R_sun, vignette=u.R_sun)
 def rhef(
     smap,
+    *,
     radial_bin_edges=None,
     application_radius=0 * u.R_sun,
     upsilon=0.35,
     method="numpy",
-    *,
-    vignette=1.5 * u.R_sun,
+    vignette=None,  # 1.5 * u.R_sun,
     progress=True,
+    animate=False,
```

Comment:
The animate will need to go.

Reply:
That's fair, thats just how I found the error.